lib/components/T3Frame/T3Frame.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <div
+    class="t3-ce-frame"
+    :class="computedClasses"
+    v-bind="$attrs"
```

Comment:
is there a need to use v-bind="$attrs" since we don't use inheritAttrs:false here ?